### ISSUE TITLE
[2.2.1] Unable scale the Sphere Mesh with OES transparent shader

### ISSUE THREAD

**Issue by downkirkland** [RenderHeads/UnityPlugin-AVProVideo#966]
**Describe the issue**
It looks like the EOS shader limits the sphere to less that 10x10, unable to scale the sphere to place other visuals inside

**Your Setup (please complete the following information):**
 - Unity version: 2020.3.17f1
 - AVPro Video version: Android 2.2.1
 - Operating system version: Mac
 - Device model: Quest 2
 - Video specs (resolution, frame-rate, codec, file size): 

**To Reproduce**
If you use any VR shader, you can scale the sphere, with EOS its limited to 10x10

**Comment by kahnivore** [comment]
Hi,

1. Are you talking about the "VR-InsideSphere-AndroidOES" shader or the "VR-InsideSphere-Transparent-AndroidOES" shader?
2. How are you trying to scale the sphere?

**Comment by downkirkland** [comment]
I tried both shaders with same issue.  I scale the sphere-applytomesh.   Yes, I can scale the game objects smaller forcing a space of 10x10 and doesn't give a true mixed-reality feel.  Unfortunately I had all of this working on the trial version with watermarks.  Once I upgraded to paid version, the video playback was choppy and felt downgraded.  I was only able to get playback working leveraging the EOS shader.
